lib/OpenQA/WebAPI/Controller/Test.pm

```diff
@@ -116,8 +116,8 @@ sub list {
     #         0;
     #     }
     # }
-    my @scheduled = $scheduled->search({blocked_by_id => undef})->all;
-    @scheduled = sort { $b->t_created <=> $a->t_created || $b->id <=> $a->id } @scheduled;
+    #  my @scheduled = $scheduled->search({blocked_by_id => undef})->all;
+    my @scheduled = sort { $b->t_created <=> $a->t_created || $b->id <=> $a->id } $scheduled->all;
     $self->stash(scheduled => \@scheduled);
```

Comment:
That's basically reverting https://github.com/os-autoinst/openQA/pull/1717/commits/e2b8233db2ade501e561b472e1650456895064a5, no? Why?

Reply:
Honestly because i found it a bit confusing. 

On staging i just scheduled 70 jobs, 65 of them were chained deps, and it was impossible to determine if openQA scheduled them or not, since they were not appearing at all in the scheduled list without this change (not even the blocked by count).

Reply:
(it's not actually reverting it.. but we display scheduled jobs anyway, with the blocked by count)

Reply:
but I find *this* confusing to see 65 scheduled jobs when I have 30 free workers. See @foursixnine's comment in the original PR - we should have a specific list of blocked jobs. But I want the 'scheduled jobs' to be those the scheduler is looking at.

Reply:
yes - but then i found myself doing what i think most of people will do: try to repost the job as you don't see them scheduled in the webui.

I can revert this commit of course. But i don't think it's a good idea as it is now - shall we separate those list then? won't be too much on the main dashboard having more lists? (if we count also the running and the finished, we end up with 4 lists then.

Reply:
@thehejik @kravciak : WDYT? makes sense for you to *not* see the jobs inside the scheduled table if they are waiting for other jobs to completion? I've just tried with slenkins tests - and that was really confusing

Reply:
Well, show the blocked ones differently in the same list - but don't just revert to the previous state

Reply:
For me it's better to have the information that jobs are scheduled and waiting for another job to finish - hiding these jobs is not a solution for me. If we can mark these blocked jobs by for eg. "orange" icon it would be an improvement.

Reply:
Something like that? 

![blocked_by](https://user-images.githubusercontent.com/2420543/43326107-40c437bc-91b8-11e8-9986-dce63a35d84e.png)

Reply:
yes, I like that ;)

Reply:
we will come back to this colour choice I'm sure, but let's go ahead with this :)